src/views/ipv/pyi-triage-select-device.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleName = 'pages.pyiTriageSelectDevice.title' | translate %}
```

Comment:
This needs to be just the pageTitleKey without being translated.

We set the key here, and then the base.njk file that we extend in every template uses the key to set the page title, and passes the key into GA4 for analytics events.